uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/ColumnRenderingStrategyMerged.java

```diff
@@ -115,7 +115,8 @@
         //Column content
         commands.add((GridRenderer.RenderBodyGridContentCommand) (rc) -> {
             if (columnRenderingConstraint.apply(rc.isSelectionLayer(), column)) {
-                final Group columnGroup = new Group().setX(x);
+                final Group columnGroup = GWT.create(Group.class);
```

Comment:
Is `GWT.create` better in terms of performance compared to `new`?

Reply:
@danielezonca 
`GWT.create` is not equivalent to `new`

Reply:
GWT.create(..) allows `GwtMockitoTestRunner` and `LienzoMockitoTestRunner` to provide mocks in Unit Tests. Use of `new` does not.